rest_framework_swagger/docgenerator.py

```diff
@@ -20,6 +20,9 @@ class DocumentationGenerator(object):
     # Response classes defined in docstrings
     explicit_response_types = dict()
 
+    def __init__(self, for_user):
```

Comment:
Since an `AnonymousUser()` ought be the same thing everywhere, I think I'd prefer:

```python
def __init__(self, for_user=AnonymousUser()):
```

Thoughts?